src/features/gallery/GalleryPostActions.tsx

```diff
@@ -60,6 +62,7 @@ export default function GalleryPostActions({ post }: GalleryPostActionsProps) {
       </Section>
       <div
         onClick={() => {
+          dispatch(setPostRead(post.post.id));
```

Comment:
This should not be needed. Post marked as read when you open the detail page

Reply:
My bad, didn't ctrl + s this file after removing